README.md

```diff
@@ -111,7 +111,10 @@ Make sure there are no spaces between the comma-separated list of paths of direc
 `bin/packs rename`
 
 ## Set packs/child_pack as a child of packs/parent_pack
-`bin/packs move_to_parent packs/child_pack packs/parent_pack `
+`bin/packs move_to_parent packs/child_pack packs/parent_pack`
+
+## Move packs/foo to the packs/bar folder
+`bin/packs move_to_folder packs/foo packs/bar`
```

Comment:
This example looks like us moving it to a pack.

> `bin/packs move_to_folder packs/foo packs/bar` where packs/bar does not contain a package.yml file

Reply:
Good call. I'll update the example to not include `packs` in the target directory.